src/views/components/package-header.html

```diff
@@ -39,7 +39,11 @@ <h1 class="package-name">{{name}}</h1>
 
 	<div class="row">
 		<div class="col-xs-12">
+			{{#if !~/type || ~/type === 'gh'}}
```

Comment:
No `type` means we're on a search [results page](https://jsdelivr-com-pr-320.herokuapp.com/?query=re), which only lists npm projects. Also to stay consistent the GitHub link should open in a new tab.